app/posts/screening-invite/2025/09/2025-08-13-invite-breast-screening-pilot-user-research.md

```diff
@@ -0,0 +1,117 @@
+
+    In this sample, concerns around not having read receipts for SMS messages were mitigated by participants being experienced in receiving SMS messages for healthcare communications.
+
+2) **Messages can be noticed even when sent via a new channel**
+Even when participants received a message in a channel they don’t normally use, most of them still noticed it and attended their appointment.
+
+    This is something to keep in mind as we roll out and potentially hear more concerns from people who are used to receiving letters. While people’s attitude may be one of concern, their behaviour can still mean they access messages and attend appointments.
+
+3) **Messages can be effective even when people’s perception or memory of them is inaccurate**
+Many of the participants in this sample did not accurately recall the type of invitation they received. Some claimed they received an email, when we didn’t send any.
+
+    This is because it can be hard to keep track of which healthcare message is for which appointment, or distinguish between different types of notification on your phone. Also people were sent SMS reminders, which can be confused with the original invitation​.
+
+    Despite this, the key point is that most people still saw their message and attended their appointment, and many were ultimately     indifferent on what type of message they received. If people are indifferent, this could make it easier to change the type of digital channel we use to send messages.
+
+4) **Most of the sample didn’t follow a link to further guidance as they didn’t need to**
+Each message had a link (or a QR code for the letter) to a gov.uk publication page. Most people in this sample didn’t follow the link because they had gone through the process before and the key information was quickly available in the message.
+
+## Attending appointments and remembering where they are
+
+We’ve found out that:
+
+1)	**The change to digital messages can impact the people who use their letter as a reminder**
+ Two people said they forgot about their appointment because they didn’t have a letter, with one of them saying they need letters because they have memory loss issues.
+
+    Letters can have the advantage of being easily visible as you put them on fridges or other places at home. For some, messages on a phone can be harder to remember in comparison and we have previously seen there can be issues relocating an NHS App message in our pre-pilot user research.
+
+
+2)	**Letters do not need to act as reminders to people with strong calendar habits**
+Most of this sample added their appointment to either a physical or digital calendar. This suggests there could be a health inequality issue to consider here. 
+
+    Our fallback system could work for those who are able to set up reminders themselves, but not for people who are less able to do this due to things like learning difficulties or digital capability and would benefit from a letter instead of an SMS. 
```

Comment:
```suggestion
Our fallback system could work for those who are able to set up reminders themselves, but not for people who are less able would benefit from letters instead of SMS.
```